Classes/Controller/EditorController.php

```diff
@@ -54,6 +62,133 @@ class EditorController extends ActionController
     protected $allowedCategories = [];
 
     /**
+     * @var IconFactory
+     */
+    protected $iconFactory;
+
+    /**
+     * BackendTemplateContainer
+     *
+     * @var BackendTemplateView
+     */
+    protected $view;
+
+    /**
+     * Backend Template Container
+     *
+     * @var BackendTemplateView
+     */
+    protected $defaultViewObjectName = BackendTemplateView::class;
+
+    /**
+     * Set up the doc header properly here
+     *
+     * @param ViewInterface $view
+     */
+    protected function initializeView(ViewInterface $view)
+    {
+        /** @var BackendTemplateView $view */
+        parent::initializeView($view);
+
+        if ($this->view->getModuleTemplate() !== NULL) {
+            $pageRenderer = $this->view->getModuleTemplate()->getPageRenderer();
+            $pageRenderer->loadRequireJsModule('TYPO3/CMS/Themes/Colorpicker');
+            $pageRenderer->loadRequireJsModule('TYPO3/CMS/Themes/ThemesBackendModule');
+
+            $extRealPath = ExtensionManagementUtility::siteRelPath('themes');
```

Comment:
extPath return path /var/www/sitefolder/. i think this won't work for css files.